internal/app/oauth/request.go

```diff
@@ -0,0 +1,116 @@
+package oauth
+
+import (
+	"time"
+
+	"github.com/caos/oidc/pkg/oidc"
+)
+
+var essentialScopes = []string{"openid", "profile", "email"}
+
+type AuthRequest struct {
+	ID            string
+	ClientID      string
+	subject       string
+	code          string
+	state         string
+	ResponseType  oidc.ResponseType
+	scopes        []string
+	RedirectURI   string
+	Nonce         string
+	CodeChallenge *oidc.CodeChallenge
+	createdAt     time.Time
+	authorizedAt  *time.Time
```

Comment:
```suggestion
	authorizedAt  *time.Time
	Domain        string
	Debug         bool
```

Inject appConfig's fields to these fields during initialization of AuthRequest